### ISSUE TITLE
build failed linux GCC 8.3

### ISSUE THREAD

**Issue by rdtor** [spell-music/csound-expression-typed#7]
src/Csound/Typed/Gui/Types.hs:28:1: error:
    Could not find module ‘Data.Colour’
    There are files missing in the ‘colour-2.3.5’ package,
    try running 'ghc-pkg check'.
    Use -v to see a list of the files searched for.
   |
28 | import Data.Colour
   | ^^^^^^^^^^^^^^^^^^
cabal: Leaving directory '/tmp/cabal-tmp-4471/csound-expression-typed-0.2.2.0'
cabal: Error: some packages failed to install:
csound-expression-typed-0.2.2.0-IRfOq67iVwvIyRPVaPPY2c failed during the
building phase. The exception was:
ExitFailure 1

**Comment by anton-k** [comment]
Thanks for report! I'll check this out

**Comment by anton-k** [comment]
I can not find your version of GHC-8.3

But I've tried with closest GHC-8.4. And it builds fine. 
My approach was: 

* init new project with `stack new csound-example`
* create this stack.yaml

```
resolver: lts-12.7

packages:
- .

extra-deps:
  - csound-expression-5.3.2
  - csound-expression-opcodes-0.0.4.0
  - csound-expression-dynamic-0.3.3
  - csound-expression-typed-0.2.2.0
  - temporal-media-0.6.3
  - data-fix-cse-0.0.2
  - wl-pprint-1.2.1
```

* add dependency csound-expression to the .cabal file of the library

* build everything ith: `stack build` - it builds fine

* go to interpreter: `stack ghci`

Then hello world works:

```
> import Csound.Base
> dac $ testDrone2 220
```

**Comment by rdtor** [comment]
Hi thx for the reply i tried installing with ghcup and now I have it
running.
